www_src/pages/project/project.jsx

```diff
@@ -43,39 +43,74 @@ var Map = React.createClass({
       });
     });
   },
+
   componentDidMount: function () {
     var el = this.getDOMNode();
     var bounding = this.refs.bounding;
     var boundingEl = bounding.getDOMNode();
-    var startX, startY, deltaX, deltaY;
+    var scaleEl = this.refs.scaler.getDOMNode();
+    var isZoom, startX, startY, startDistance, deltaX, deltaY, currentZoom;
     var didMove = false;
+    var ZOOM_SENSITIVITY = 300;
 
     el.addEventListener('touchstart', (event) => {
       didMove = false;
       startX = event.touches[0].clientX;
       startY = event.touches[0].clientY;
       boundingEl.style.transition = 'none';
+      scaleEl.style.transition = 'none';
```

Comment:
There is no need to work with DOM nodes other than to get the intial bounds. Instead of setting the .style property manually, update the properties via setState() and and then let `render()` handle the diffing.